src/components/WebglBox.tsx

```diff
@@ -14,10 +14,13 @@ const WebglRoot = styled.div`
 
 type WebglProps = {
   userId: string;
-  experienceData: ExperienceData;
 };
 
-function WebglBox({ userId, experienceData }: WebglProps): JSX.Element {
+function WebglBox({ userId }: WebglProps): JSX.Element {
```

Comment:
userID is already passed as context so it doesn't need to be passed as props to webglbox

Reply:
Taken care of , thank you